code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -37,9 +37,13 @@
 
 /obj/item/encryptionkey/amplification
 	name = "amplification module key"
-	desc = "An amplification module key for a radio headset. It will enable the \"Loud mode\" ability on any headset it is inserted into."
 	amplification = TRUE
 
+// Makes sure neither channel messages show up.
+/obj/item/encryptionkey/amplification/Initialize()
+	. = ..()
+	desc = "An amplification module key for a radio headset. It will enable the \"Loud mode\" ability on any headset it is inserted into."
```

Comment:
This feels dodgy

Reply:
wdym

Reply:
It does but otherwise the parent proc botches up the description